tee-worker/app-libs/stf/src/trusted_call.rs

```diff
@@ -142,6 +147,10 @@ pub enum TrustedCall {
 	send_erroneous_parentchain_call(Identity),
 	#[codec(index = 25)]
 	maybe_create_id_graph(Identity, Identity),
+	#[codec(index = 25)]
+	update_token_staking_amounts(Identity, Vec<(AccountId, Balance)>),
```

Comment:
I think this call can be removed from the pool due to limits for example.
We should make sure there is no way to delay / prevent execution by regular rpc clients.

Reply:
Same here. After the discussions, we are going back to putting all the logic in the event_handler